pensjon-brevbaker/src/main/kotlin/no/nav/pensjon/brev/template/TemplateModelSpecificationFactory.kt

```diff
@@ -13,7 +14,7 @@ class TemplateModelSpecificationFactory(val from: KClass<*>) {
     private val toProcess = mutableListOf<KClass<*>>()
 
     fun build(): TemplateModelSpecification =
-        if (from.objectInstance == Unit || from.objectInstance == EmptyBrevdata) {
+        if (from.objectInstance == Unit || from.objectInstance in setOf(EmptyBrevdata, EmptyRedigerbarBrevdata)) {
```

Comment:
Hmm... Jeg tror kanskje vi må fikse litt i Skribenten for å støtte en mal med EmptyBrevData. 
Se `findSaksbehandlerValgTypeName` i ModelEditor.tsx.